mne/viz/utils.py

```diff
@@ -2510,20 +2516,23 @@ def _check_cov(noise_cov, info):
 
 
 def _set_title_multiple_electrodes(title, combine, ch_names, max_chans=6,
-                                   all=False, ch_type=None, ):
+                                   all=False, ch_type=None):
     """Prepare a title string for multiple electrodes."""
     if title is None:
         title = ", ".join(ch_names[:max_chans])
-        if ch_type is not None:
-            ch_type = " " + ch_type[0].upper() + ch_type[1:]
+        ch_type = _channel_type_prettyprint.get(ch_type, ch_type)
+        if ch_type is None:
+            ch_type = ""
+        elif len(ch_names) > 1:
+            ch_type += "s"
```

Comment:
Does this also work if you have "Misc. sensors" in the dict?

Reply:
Ah, no, you're right, it doesn't.
But why capitalise it? "Average of 30 misc. sensors" seems just right to me. (Or unabbreviated, although I think we probably have too many characters on the screen already.)

Reply:
Yes, forget about capitalization then. I'd prefer miscellaneous, but if screen real estate is really an issue then feel free to use misc.

Reply:
Well, not so much estate, but it becomes visually very crowded. Right above it it reads "AFz, Fz, FCz, Cz, Pz, POz, ...", that's a lot of ink already.

Reply:
Right, I just *visually* don't like an abbreviation in the title. Maybe a more accurate term would be "other sensors", but this goes beyond this PR.

Reply:
> Maybe a more accurate term would be "other sensors"

"other" doesn't work here actually. Other than what?

>I just visually don't like an abbreviation in the title

Then you're already suffering because the line above ends with an ellipsis :)

Reply:
Other than what we have defined in the list. But you're right, it doesn't work here. The ellipsis is fine, it only hurts when I read "misc. sensors" :smile:.

Reply:
Ok, I'll spell it out :) Mayyyybe I was just too lazy to look up the spelling.

Reply:
Hehe 🤣 😆